### DISCUSSION TITLE
Student Verfication Failed

### DISCUSSION THREAD

**Discussion by ghost** [community/community#122582] [GitHub Education]
### Hi everyone,

What I uploaded for proof is what from the school letter. But dk why, the system keep rejecting my submission

**Comment by jordan-k-johnson** [comment]
According to [a similar discussion](https://github.com/orgs/community/discussions/59018):

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio. The verification process looks at your GitHub profile to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Is your name in non-english? When you update your profile information please log out and log back into GitHub before re-applying.

**Comment by Omega403** [comment]
Hi,
if you mean verifying on GitHub as a student to obtain the advantages, I can tell you that I created this account recently by doing the verification and initially it also didn't accept my university's enrollment certificate,I solved it by simply uploading a different version of the certificate (simply renaming it) . 

Then make sure you have added your university/institutional credentials to your account before verifying, 
by this I mean add your institutional email, also it seems to me that you had to accept a series of security parameters such as 2FA... 

try to do this, in case of success you have to wait a few days before the benefits are activated
